src/tutorial/furthertopics/message-triggers.rst

```diff
@@ -0,0 +1,439 @@
+         -               long_forecasting_task => different_weather_task 
+         +               long_forecasting_task:update1 =>  another_weather_task
+         +               long_forecasting_task:update2 => different_weather_task 
+                         long_forecasting_task[-P2M] => long_forecasting_task
+                     """
+
+      This completes our ``suite.rc`` file.
+
+      Our final suite should look like this:
+
+      .. spoiler:: Solution warning
+
+         .. code-block:: cylc
+
+            [cylc]
+            UTC mode = True
+
+            [meta]
+            title = "test suite to demo message triggers"
+
+            [scheduling]
+                initial cycle point = 2019-06-27T00Z
+                final cycle point = 2019-10-27T00Z
+                
+                [[dependencies]]
+
+                    [[[P2M]]]
+                        graph = """
+                            long_forecasting_task:update1 =>  another_weather_task
+                            long_forecasting_task:update2 => different_weather_task 
+                            long_forecasting_task[-P2M] => long_forecasting_task
+                        """
+
+            [runtime]
+
+                [[long_forecasting_task]]
+                    script = """
+                        sleep 2
+                        random.sh
+                        cylc message -- "${CYLC_SUITE_NAME}" "${CYLC_TASK_JOB}" \
+                            "Task partially complete, report ready to view"
+                        sleep 2
+                        random.sh
+                        cylc message -- "${CYLC_SUITE_NAME}" "${CYLC_TASK_JOB}" \
+                            "Task partially complete, report updated"
+                        sleep 2
+                        random.sh
+                    """
+
+                    [[[outputs]]]
+                        update1 = "Task partially complete, report ready to view"
+                        update2 = "Task partially complete, report updated"
+
+                [[another_weather_task, different_weather_task]]
+                    script = sleep 1
+
+   #. **Validate the suite.**
+
+      It is a good idea to check that our ``suite.rc`` file does not have any
+      configuaration issues. 
+
+      Run `cylc validate` to check for any errors:
+
+      .. code-block:: bash      
+
+          cylc validate .
+   
+   #. **Run the suite.**
+
+      Now we are ready to run our suite. Open the Cylc GUI by running the
+      following command:
+
+      .. code-block:: bash
+
+         cylc gui message-triggers &
+
+      Run the suite either by pressing the play button in the Cylc GUI or by
+      running the command:
+
+      .. code-block:: bash
+
+         cylc run message-triggers
+
+      Your suite should now run, the tasks should succeed. 
+
+   #. **Inspect the work directory.**
+
+      We can now check for our report outputs. These should appear in the 
```

Comment:
```suggestion
      You can now check for your report outputs. These should appear in the 
```

This is a matter of subjective style I guess, but you're mostly refering to "we" and "our", but occasionally "Your" (line 356).  Personally I think "you" and "your" is a bit more direct and snappy.